esp-temperature-display/ESP32_Support.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ESP32_Support_H
+#define ESP32_Support_H
+
+#if defined(ESP32)
+
+#include <WiFi.h>    // Built in
+#include <ESPmDNS.h> // Built in
+
+// For ESP_DoubleResetDetector
+#ifndef LED_BUILTIN
+#define LED_BUILTIN       2         // Pin D2 mapped to pin GPIO2/ADC12 of ESP32, control on-board LED
+#endif
+
+#define LED_OFF     LOW
+#define LED_ON      HIGH
+
+#include <AsyncTCP.h> // From Library Manager, version 1.1.4
+
+String chipID = ESP.getChipModel();
```

Comment:
Since the chip ID is used for unique identification this should be the Mac Address instead. The model would just be "ESP32-WROOM32" for example.

Fun fact the ChipID on ESP8266 is also just the mac address.

Reply:
Will make this change :D Thanks!